### ISSUE TITLE
Using Orthofinder to find horizontal gene transfer

### ISSUE THREAD

**Issue by scubapenn** [davidemms/OrthoFinder#316]
Would orthofinder be able to find fin orthologs one of which which was acquired via HGT?

**Comment by davidemms** [comment]
Yes, in many cases it will be able to as OrthoFinder doesn't rely on the gene tree and species tree agreeing in order to identify orthologs. Additionally, if you have high enough species sampling then OrthoFinder should also be able to identify that the gene is not where it is expected in the gene tree, identify that it is a putative HGT and identify that the related genes are xenologs and put these relationships in the Putative_Xenologs/ directory.

All the best
David

**Comment by mtnouchi** [comment]
Hello, 

I have a question for this topic.
As far as I understand, OrthoFinder uses DLCpar for reconciliation and DLCpar only models duplication, loss and deep coalescence but not transfer. 

How does OrthoFinder infer HGTs when DLCpar does not assume transfer events unlike other reconciliation algorithms such as ranger-dtl?

Regards,

**Comment by davidemms** [comment]
Hi

DLCpar was only used a few years ago, in an early version of the orthologs from gene trees functionality. It wasn't accurate enough and so was replaced with a new algorithm, described in the latest paper: <https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1832-y> 

The current version identifies 'Phylogenetically Misplaced Genes" as an additional step. At the moment I would still call this prototype functionality. Its purpose is to avoid missing true orthologs within the gene trees because of genes that appear to be out of place in the gene tree, given knowledge of the species tree. In a number of cases I've looked at the source of many of these genes is contamination, and thus identifying them is useful. If you are looking for HGTs then these genes are a good place to start but that's not its main goal

All the best
David
